site2/docs/io-flume-source.md

```diff
@@ -0,0 +1,21 @@
+---
+id: io-flume-source
+title: Flume source connector
+sidebar_label: Flume source connector
+---
+
+The Flume source connector pulls messages from logs to Pulsar topics.
+
+## Configuration
+
+The configuration of Flume source connector has the following parameters.
+
+### Parameter
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+`name`|String|true|"" (empty string)|Name of the agent
+`confFile`|String|true|"" (empty string)|Configuration file
+`noReloadConf`|Boolean||false|Whether to reload configuration file if changed
```

Comment:
@tuteng Is `noReloadConf` required or optional? I don't find the information in https://github.com/apache/pulsar/blob/master/pulsar-io/flume/src/main/java/org/apache/pulsar/io/flume/FlumeConfig.java

Reply:
optional